apps/test/unit/templates/lessonOverview/LessonOverviewTest.js

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import {shallow} from 'enzyme';
 import {expect} from '../../../util/reconfiguredChai';
 import LessonOverview from '@cdo/apps/templates/lessonOverview/LessonOverview';
+import {sampleActivities} from '../../../../test/unit/lib/levelbuilder/lesson-editor/activitiesTestData';
```

Comment:
nit: here and below, you shouldn't have to go up quite so many directories to find this: `import {sampleActivities} from '../..//lib/levelbuilder/lesson-editor/activitiesTestData';` but if this helps you or your IDE somehow then it seems ok to leave.